conda/logic.py

```diff
@@ -331,7 +331,74 @@ def LB_Preprocess_(self, equation):
         equation = sorted(equation)
         return equation, offset
 
-    def LinearBound_(self, equation, lo, hi, preprocess, polarity):
+    def Sorter_(self, equation, polarity=None):
```

Comment:
This new method here is the cause for the coverage decreasing. At some point we should write tests for it, but that's not as urgent right now as making sure we meet the Friday deadline.